TODO/JS Exercises mix/index2.html

```diff
@@ -0,0 +1,28 @@
+<!DOCTYPE html>
+<html>
+    <head>
+        <script>
+            function AddNewItem() {
+                let number = 4;
```

Comment:
you should nor "hard code" this number, you should get it directly from the DOM.